data/rbac_catalog_seed.yml

```diff
@@ -1,139 +1,10 @@
 ---
-resource_definition1: &AdminResourceDefinition
```

Comment:
@mkanoor since these aren't needed in seeding, were they just being ignored before?

Reply:
@syncrou The seeding code checks if the role exists, if it exists which will be true since its a System role (seeded by RBAC service) it will not get re-created. So it was always noise, that seeding yaml existed before the RBAC service did the seeding of System roles. The System roles are defined here in the RBAC service
https://github.com/RedHatInsights/insights-rbac/blob/7903c0bedce45cc4f01503fb4cee5c6dae802370/rbac/management/role/definitions/catalog.json#L4